wee_slack.py

```diff
@@ -3117,6 +3160,18 @@ def process_channel_marked(message_json, eventrouter, team, channel, metadata):
 process_mpim_marked = process_channel_marked
 
 
+def process_thread_marked(message_json, eventrouter, team, channel, metadata):
+    subscription = message_json.get("subscription", {})
+    ts = subscription.get("last_read")
+    thread_ts = subscription.get("thread_ts")
+    channel = team.channels.get(subscription.get("channel"))
+    if ts and thread_ts and channel:
+        thread_channel = channel.thread_channels.get(thread_ts)
```

Comment:
You need to create a `SlackTS` of the `thread_ts` before you look it up in `thread_channels`.